aws_google_auth/google.py

```diff
@@ -196,12 +196,8 @@ def do_login(self):
     def check_extra_step(response):
         extra_step = response.find(text='This extra step shows that it’s really you trying to sign in')
         if extra_step:
-            print(extra_step)
-            msg = response.find(id='contactAdminMessage')
-            if msg:
-                raise ValueError(msg.text)
-            else:
-                raise ValueError(response)
+            if response.find(id='contactAdminMessage'):
+                raise ValueError(response.find(id='contactAdminMessage').text)
```

Comment:
Should there still be an `else` here on a page missing the `contactAdminMessage` element? 
Some way for us to troubleshoot what is different on that error page that we need to process.

Reply:
The idea here is to unblock Google Prompt, yet print the message if GSuite Administrator rules are violated (like lack of MFA when is required). 

I think, in the long term something similar to debug mode (dump all responses and SAML assertion) would be useful.

Reply:
Makes sense, let's ship this and look further at the debug workflow.

Reply:
@stevemac007 what version of the tool is this shipped in? Want to verify if I'm running a version including this fix before I report an issue.